packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -85,9 +79,32 @@ export const createHandler = async ({ input, ctx }: CreateOptions) => {
 
   const t = await getTranslation(ctx.user.locale ?? "en", "common");
   const availability = getAvailabilityFromSchedule(DEFAULT_SCHEDULE);
+  let isOrganizationConfigured = false;
 
   if (check === false) {
-    if (IS_CALCOM) await vercelCreateDomain(slug);
+    // eslint-disable-next-line turbo/no-undeclared-env-vars
+    if (process.env.VERCEL) {
+      // We only want to proceed to register the subdomain for the org in Vercel
+      // within a Vercel context
+      isOrganizationConfigured = await vercelCreateDomain(slug);
+    } else {
+      // Otherwise, we proceed to send an administrative email to admins regarding
+      // the need to configure DNS registry to support the newly created org
+      const instanceAdmins = await prisma.user.findMany({
+        where: { role: UserPermissionRole.ADMIN },
+        select: { email: true },
+      });
+      if (instanceAdmins.length) {
+        await sendAdminOrganizationNotification({
+          instanceAdmins,
+          orgSlug: slug,
+          ownerEmail: adminEmail,
+          t,
+        });
+      } else {
+        console.warn("Organization created: subdomain not configured and couldn't notify adminnistrators");
```

Comment:
No Vercel, no email sent to admins, so we warn in console and we mark the org as DNS missing until admins mark it as configured from settings